src/sds.c

```diff
@@ -1576,6 +1599,32 @@ int sdsTest(int argc, char **argv, int flags) {
         test_cond("sdsrezie() crop strlen", strlen(x) == 4);
         test_cond("sdsrezie() crop alloc", sdsalloc(x) == 4);
         sdsfree(x);
+
+        /* Test sds_have_block_chars. */
+        x = sdsnew("char\r");
+        test_cond("sds_have_block_chars() with char \\r", sds_have_block_chars(x) == 1);
+        sdsfree(x);
+        x = sdsnew("char\n");
+        test_cond("sds_have_block_chars() with char \\n", sds_have_block_chars(x) == 1);
+        sdsfree(x);
```

Comment:
no need to test each and every char.
but i guess once you either eliminate this function, or make it more generic, and pass the list of chars from outside, this will either be gone, or test will be made much shorter.